recipe/meta.yaml

```diff
@@ -33,8 +35,10 @@ test:
 about:
   home: https://github.com/pypa/packaging
   license: Apache 2.0 or BSD 2-Clause
+  license_file: LICENSE
```

Comment:
Would be good if we could include both license files, but that is not so easy actually. Let's leave it for later.

xref: https://github.com/conda/conda-build/issues/1247